WalletWasabi.Packager/Program.cs

```diff
@@ -367,10 +384,34 @@ private static async Task PublishAsync()
 					continue;
 				}
 
-				ZipFile.CreateFromDirectory(currentBinDistDirectory, Path.Combine(BinDistDirectory, $"Wasabi-osx-{VersionPrefix}.zip"));
+				var zipFileName = target.Contains("arm64")
+					? $"_WasabiToMac-osxarm64-{VersionPrefix}.zip"
+					: $"_WasabiToMac-osx-{VersionPrefix}.zip";
+
+				var zipFilePath = Path.Combine(BinDistDirectory, zipFileName);
+
+				ZipFile.CreateFromDirectory(currentBinDistDirectory, zipFilePath);
 
 				await IoHelpers.TryDeleteDirectoryAsync(currentBinDistDirectory).ConfigureAwait(false);
 				Console.WriteLine($"Deleted {currentBinDistDirectory}");
+
+				try
+				{
+					Console.WriteLine($"Trying to copy unsigned zip file to removable.");
+					var driveList = DriveInfo.GetDrives().Where(d => d.DriveType == DriveType.Removable);
+					if (!driveList.Any())
+					{
+						Console.WriteLine($"No removable drive found - finish.");
+					}
+
+					var drive = driveList.Single();
+					var targetFilePath = Path.Combine(drive.Name, zipFileName);
+					File.Move(zipFilePath, targetFilePath, overwrite: true);
```

Comment:
This might be useful to avoid confusion when one has multiple USB disks in:

```suggestion
					var driveList = DriveInfo.GetDrives().Where(d => d.DriveType == DriveType.Removable);
					if (!driveList.Any())
					{
						Console.WriteLine($"No removable drive found - finish.");
					}

					DriveInfo drive = driveList.Single();
					string targetFilePath = Path.Combine(drive.Name, zipFileName);

					Console.WriteLine($"Trying to copy unsigned zip file to removable ('{targetFilePath}').");
					File.Move(zipFilePath, targetFilePath, overwrite: true);
```

i.e. one then knows immediately where the file was copied. Also the console line says "copy" but we use `File.Move`...? Should we copy or move?

Reply:
I prefer Move - I don't want to clean up afterward.